### PR TITLE
Labcoat storage change and medical sprite changes

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

Changes the lab coat to function storage wise like a medical storage module.
Ports the **old** TG lab coat sprites, not the new ones that were so contentious.
Improves the medical scrubs sprite, making sure it covered the whole human mob whilst improving their overall look.
MDs now also start with a lab coat.
The researcher uniform and lab coat have been resprited.
Removed the unused solid color lab coats.
Removed the armored lab coat due to it's broken sprite and near instant abandonment by researchers.
Moved the snowsuits from labcoat.dm into jobs.dm for better organization.

## Why It's Good For The Game

Right now lab coats can only store two items small items inside their storage. By changing what they can hold to match the medical storage module, the lab coat becomes much more competitive with armor shipside and less likely to be replaced.

New lab coat storage means improved lab coat sprites, these are the old TG lab coat sprites before the new contentious ones were added.
00doktor!
You can also see the new researcher uniform (also ported from TG) and improved scrubs (they really didn't change much, just a few more pixels).

Because lab coats are now actually useful it makes sense for MDs to have them.
The solid color lab coats didn't show up anywhere else in code or maps.
Armored lab coat sprite doesn't sit right (or in my opinion look that good). And balance wise it's too slow for shipside, and not protective, storage efficient, or modular enough for groundside, leaving it the eternal role of being left on the floor until a roomba eats it. Therefore it also goes in favor of a more standard lab coat.
Why snowsuits were in the lab coat file of all places is beyond me, they have been moved to a better location.

## Changelog
:cl:
balance: Lab coats now have the same storage as a medical storage module.
imageadd: New sprites for all lab coats
imageadd: New researcher uniform and improved scrubs sprite.
/:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
mfw those palletes

File: code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -31,195 +18,114 @@
 		if(!usr.canmove || usr.stat || usr.restrained())
 			return 0
 
-		//Why???
```
Comment (TiviPlus) [inline code/modules/clothing/suits/labcoat.dm:34]: dont remove this why comment it calls out this massive garbage

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Fix your cl

@Kuro020 review

**Comment by Arctrooper8** [comment]
CL fixed and why comment readded!

**Review by Kuro020 — APPROVED** [review]

**Pull request merged**